<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>operate</class>
 <widget class="QMainWindow" name="operate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget
{
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="come_late_text">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>15</y>
      <width>105</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 35px &quot;微软雅黑&quot;;
color: rgb(112, 112, 112);
</string>
    </property>
    <property name="text">
     <string>迟到：</string>
    </property>
   </widget>
   <widget class="QLabel" name="leave_early_text">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>15</y>
      <width>105</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 35px &quot;微软雅黑&quot;;
color: rgb(112, 112, 112);
</string>
    </property>
    <property name="text">
     <string>早退：</string>
    </property>
   </widget>
   <widget class="QToolButton" name="search">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rsc.qrc">
      <normaloff>:/img/find.png</normaloff>:/img/find.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>240</width>
      <height>180</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="today">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>225</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rsc.qrc">
      <normaloff>:/img/tody.png</normaloff>:/img/tody.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>240</width>
      <height>180</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="delinfo">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>420</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rsc.qrc">
      <normaloff>:/img/del.png</normaloff>:/img/del.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>240</width>
      <height>180</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="out">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>607</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rsc.qrc">
      <normaloff>:/img/out.png</normaloff>:/img/out.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>147</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="leave_early">
    <property name="geometry">
     <rect>
      <x>401</x>
      <y>80</y>
      <width>390</width>
      <height>460</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rsc.qrc">:/img/list.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="come_late">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>390</width>
      <height>460</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rsc.qrc">:/img/list.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="num">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>345</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rsc.qrc">:/img/num_in.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>570</y>
      <width>345</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rsc.qrc">:/img/name_in.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="num_Edit">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>591</y>
      <width>125</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25px &quot;微软雅黑&quot;;
color: rgb(112, 112, 112);</string>
    </property>
    <property name="text">
     <string>请输入学号</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_Edit">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>591</y>
      <width>125</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25px &quot;微软雅黑&quot;;
color: rgb(112, 112, 112);</string>
    </property>
    <property name="text">
     <string>请输入姓名</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="list1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>330</width>
      <height>400</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 25px &quot;微软雅黑&quot;;
color: rgb(112, 112, 112);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoScrollMargin">
     <number>10</number>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashDotLine</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>张三</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>李四</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>7：00</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>8：00</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="list2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>330</width>
      <height>400</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 25px &quot;微软雅黑&quot;;
color: rgb(112, 112, 112);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoScrollMargin">
     <number>10</number>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashDotLine</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>张三</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>李四</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>7：00</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>8：00</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
